### PR TITLE
Update register-devices-self.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about how to restrict running scripts on the computer.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3412

File: microsoft-365/managed-desktop/get-started/register-devices-self.md [hunk 1]
```diff
@@ -43,6 +43,9 @@ You can use the [Get-WindowsAutoPilotInfo.ps1](https://www.powershellgallery.com
 2.	Run `Install-Script -Name Get-WindowsAutoPilotInfo`
 3.	Run `powershell -ExecutionPolicy Unrestricted Get-WindowsAutoPilotInfo -OutputFile <path>\hardwarehash.csv`
 
+> [!NOTE]
+> Once the Script has been successfully executed, you can run `powershell -ExecutionPolicy restricted` to restrict running scripts on the computer.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-started/register-devices-self.md]:
```suggestion
> Once the script has been successfully executed, you can run `powershell -ExecutionPolicy restricted` to restrict the running of scripts on the computer.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@jaimeo Please merge this PR. Thanks.

**Pull request merged**